android/src/main/java/org/mozilla/experiments/nimbus/NimbusClientInterface+.kt

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is where the Rust API starts. We mock it out as extension variables to (a dummy) `NimbusClientInterface`.